src/hardware/addon-protocols/rs232-usb.md

```diff
@@ -231,6 +231,25 @@ Issued by the GO device on receipt of Binary Data of 256 bytes or more from the
 | Checksum | 2 | 3+x |
 | ETX (0x03) | 1 | 5+x |
 
+### Msg Type 0x26: Protobuf data packet
```

Comment:
It seems that in the MR, there is no message 0x26, is that newly added? where can I find in the code?

Reply:
it is newly added by Norman. 
https://docs.google.com/document/d/1SdDf78K7OUVpMWRelSwTW_s3kfAgGiWAhRXUhpTKMdg/edit#heading=h.i74exiijwj50